public/js/imagesloaded.pkgd.min.js

```diff
@@ -0,0 +1,156 @@
+/*!
```

Comment:
Seeing there are a few third-party JavaScript files with the new theme (which is perfectly fine!), let's make a distinction between "our code" and "third-party code".

My suggestion here is to move all third party code in `public/js/vendor/` (a new vendor folder). This way it'll be easier to maintain and keep track of it. :-)

Reply:
Check